app/src/main/res/values/strings.xml

```diff
@@ -62,5 +62,25 @@
     <string name="about_Pp">Privacy Policy</string>
     <string name="about_Oss">Used Open Source Libraries</string>
     <string name="title_activity_os_lib">About</string>
+    <string name="title_activity_feedback">FeedbackActivity</string>
+
+
+    <string-array name="feedback_topics">
+        <item>Suggest new features</item>
+        <item>Report issues with the iOS app</item>
```

Comment:
Remove "Report issues with the iOS app". See my comment on https://assembly.com/kanshu/bounties/130